pkg/infra/log/ctx_log.go

```diff
@@ -0,0 +1,36 @@
+package log
+
+import "context"
+
+type contextualArgsContextKey struct{}
+
+var contextualArgsKey = contextualArgsContextKey{}
+
+type contextualArguments struct {
+	args []interface{}
+}
+
+// WithContextualArgs adds the key/value arguments to the provided context.
+func WithContextualArgs(ctx context.Context, args ...interface{}) context.Context {
```

Comment:
actually the function is not using log, it takes a context and add the pair into the context, maybe we can leave it in the tracing package instead

Reply:
Related to below I guess?

> 1. Make [tracer.Start](https://github.com/grafana/grafana/blob/7700b529f951528e6fb8405b7fccba9fd01e26a1/pkg/infra/tracing/opentelemetry_tracing.go#L37) responsible for calling `log.WithContextualArgs(ctx, "traceID", traceID)`.